operator/pkg/apis/istio/v1alpha1/validation/validation.go

```diff
@@ -135,6 +128,18 @@ func checkDeprecatedSettings(iop *v1alpha1.IstioOperatorSpec) (util.Errors, []st
 		{"AddonComponents.tracing.Enabled", "the samples/addons/ deployments", false},
 		{"AddonComponents.kiali.Enabled", "the samples/addons/ deployments", false},
 		{"AddonComponents.prometheus.Enabled", "the samples/addons/ deployments", false},
+		{
+			"Values.telemetry.v2.stackdriver.logging", "Values.telemetry.v2.stackdriver.outboundAccessLogging and Values.telemetry.v2.stackdriver.inboundAccessLogging",
```

Comment:
In line 131 to 133,
The "new" field in the deprecatedSettings structure is set as "Values.telemetry.v2.stackdriver.outboundAccessLogging and Values.telemetry.v2.stackdriver.inboundAccessLogging", which seems to be incorrect because outboundAccessLogging and inboundAccessLogging are removed by this PR.

Reply:
Thanks! I fixed this